src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/bitbucket/client/model/CodeInsightsAnnotation.java

```diff
@@ -26,7 +26,7 @@
 public class CodeInsightsAnnotation {
     @JsonProperty("line")
     private final int line;
-    @JsonProperty("message")
```

Comment:
@mc1arke unfortunately since this change PR decoration fails with the following stracktrace for me:
```
sonarqube    | com.github.mc1arke.sonarqube.plugin.ce.pullrequest.bitbucket.client.BitbucketException: The field 'message' is required
sonarqube    | 	at com.github.mc1arke.sonarqube.plugin.ce.pullrequest.bitbucket.client.BitbucketServerClient.validate(BitbucketServerClient.java:189)
sonarqube    | 	at com.github.mc1arke.sonarqube.plugin.ce.pullrequest.bitbucket.client.BitbucketServerClient.uploadAnnotations(BitbucketServerClient.java:114)
sonarqube    | 	at com.github.mc1arke.sonarqube.plugin.ce.pullrequest.bitbucket.BitbucketPullRequestDecorator.updateAnnotations(BitbucketPullRequestDecorator.java:153)
sonarqube    | 	at com.github.mc1arke.sonarqube.plugin.ce.pullrequest.bitbucket.BitbucketPullRequestDecorator.decorateQualityGateStatus(BitbucketPullRequestDecorator.java:98)
sonarqube    | 	at com.github.mc1arke.sonarqube.plugin.ce.pullrequest.PullRequestPostAnalysisTask.finished(PullRequestPostAnalysisTask.java:160)
```

The Code Insights API expects the field to be called message: https://docs.atlassian.com/bitbucket-server/rest/7.7.0/bitbucket-code-insights-rest.html#idp12

Maybe an additional field for summary is needed?

Reply:
I think there is a difference between Cloud and Server that went unnoticed in this PR - the Cloud API description lists the summary property, but not message property: https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Bworkspace%7D/%7Brepo_slug%7D/commit/%7Bcommit%7D/reports/%7BreportId%7D/annotations